File: JS-Advanced-October-2016/15. Modules-Babel-RequireJS/Lab/Inheritance/person.js
"use strict";

import Entity from "./entity";

export default class Person extends Entity {
    constructor(name, phrase, dog) {
        super(name);

        this.phrase = phrase;
        this.dog = dog;
    }

    saySomething() {
        return `${this.name} says: ${this.phrase}${this.dog.name} barks!`;
    }
}

File: Databases-Advanced-Entity-Framework-October-2017/02. Defining-Classes-Exercises/Exercises/Opinion-Poll/Person.cs
public class Person
{
    public string Name;
    public int Age;

    public Person(string name, int age)
    {
        this.Name = name;
        this.Age = age;
    }

    public Person(int age)
        : this("No name", age)
    {
    }

    public Person()
        : this("No name", 1)
    {
    }

    public override string ToString()
    {
        return $"{this.Name} - {this.Age}";
    }
}